app/src/test/java/com/github/alfabravo2013/readyforexams/domain/login/LoginUseCaseTest.kt

```diff
@@ -0,0 +1,93 @@
+package com.github.alfabravo2013.readyforexams.domain.login
+
+import com.github.alfabravo2013.readyforexams.util.Result
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.verify
+import org.junit.jupiter.api.DisplayName
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.Assertions.assertTrue
+
+internal class LoginUseCaseTest {
+
+    @Test
+    @DisplayName("Given invalid email and any password Then Result.Failure")
+    fun invalidEmail() {
+        val repository = mockk<LoginRepository>()
+        every { repository.login(any(), any()) } returns Result.Failure()
+        val loginUseCase = LoginUseCase(repository)
```

Comment:
there is a @Before annotation and you can create a fun setUp() annotated with @Before to init the class at test only once. Also the loginRepository should be top class declaration as mock and not mocked inside all your tests.